main/UseCases/InterviewMaker.java

```diff
@@ -72,8 +74,9 @@ public Object storePlayerChoice(Object options){
      * corresponding response from the response tree in GameMaker.
      */
     // needs structure of the response tree?
-    public String internChoiceResponse(Object playerChoice, InterviewIntern intern){
-        return this.currentHRSystem.getInternChoiceResponse(playerChoice, intern);
+    public String displayInternChoiceResponse(String input){
+        this.hasSaidResponse = true;
```

Comment:
Why is hasSaidResponse set to true in this method? Does this mean once this method is called ONCE the interview will end??

Reply:
DisplayInterviewChoiceResponse still called during the first phase of the interview process (where we are displaying the response associated with the player’s choice). hasSaidResponse records is if the intern has already given a response (i.e. at the end of the first around of questioning), and it’s initialized to false in InterviewMaker bc an interview intern automatically hasn’t given a response yet (and in interviewLevel the if condition is if hasSaidResponse is false that’s when we ask interviewMaker to displayInternChoiceResponse and then hasSaidResponse is now updated to true)

Reply:
Yea but then in interviewLevel you prompt to ask to hire the intern immediately after hasSaidResponse is set to true